index.html

```diff
@@ -1,183 +1,287 @@
                 <h3>Language</h3>
             </div>
             <ul>
-                <li><p>Javascript</p></li>
-                <li><p>Ruby</p></li>
-                <li><p>Html</p></li>
-                <li><p>Css</p></li>
-            </ul>            
+                <li>
+                    <p>Javascript</p>
+                </li>
+                <li>
+                    <p>Ruby</p>
+                </li>
+                <li>
+                    <p>Html</p>
+                </li>
+                <li>
+                    <p>Css</p>
+                </li>
+            </ul>
         </section>
-        
+
         <section class="languages">
             <div class="diamond">
                 <div class="square"></div>
                 <h3>Frameworks</h3>
             </div>
             <ul>
-                <li><p>Bootstrap</p></li>
-                <li><p>Ruby on Rails</p></li>
-                <li><p>RSPec</p></li>
-                <li><p>CapyBara</p></li>
-                <li><p>Selenium</p></li>
-            </ul>            
+                <li>
+                    <p>Bootstrap</p>
+                </li>
+                <li>
+                    <p>Ruby on Rails</p>
+                </li>
+                <li>
+                    <p>RSPec</p>
+                </li>
+                <li>
+                    <p>CapyBara</p>
+                </li>
+                <li>
+                    <p>Selenium</p>
+                </li>
+            </ul>
         </section>
-        
+
         <section class="languages">
             <div class="diamond">
                 <div class="circle"></div>
                 <h3>Skills</h3>
             </div>
             <ul>
-                <li><p>CodeKit</p></li>
-                <li><p>GitHub</p></li>
-                <li><p>Codepen</p></li>
-                <li><p>GitLab</p></li>
-                <li class="last"><p>Terminal</p></li>
-            </ul> 
+                <li>
+                    <p>CodeKit</p>
+                </li>
+                <li>
+                    <p>GitHub</p>
+                </li>
+                <li>
+                    <p>Codepen</p>
+                </li>
+                <li>
+                    <p>GitLab</p>
+                </li>
+                <li class="last">
+                    <p>Terminal</p>
+                </li>
+            </ul>
         </section>
+
+        <footer>
+            <form action="https://formspree.io/f/myyogbpn" method="post">
+                <h3 class="more-info">
+                    I'm always interested in hearing about new projects, so if you'd
+                    like to chat please get in touch.
+                </h3>
+                <ul class="items-form">
+                    <li class="li">
+                        <input type="text" placeholder="Full name" maxlength="30" required name="fullname" />
+                        <div class="vector-contact"></div>
+                    </li>
```

Comment:
- [ ] [OPTIONAL] Your design is really accurate very well done 👍. Although here you can see that there is supposed to be some inline padding that pushes the placeholder text inside. Implementing this will make your design match better.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144276469-85b30cf9-5f92-4ac2-8518-432f5aa2ef0b.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144276505-8fcb74b8-cab0-420f-80dc-a6ddbf47ef0d.png)

Reply:
Thanks for the feedback, I will make the change :smile: